docs/how-to-guides/index.rst

```diff
@@ -0,0 +1,15 @@
+.. _how_to_guides:
+
+How-to guides
```

Comment:
@rokroskar Do we need to include this in the main docs somehow?

Reply:
the idea was to create a link there to this section. renku-python is now a submodule of the main renku repo for this purpose. Please make an issue for it in the renku repo for now.

Reply:
https://github.com/SwissDataScienceCenter/renku/issues/2379